docs/source/getting-started/installation.rst

```diff
@@ -92,6 +94,20 @@ Note: if you encounter *Authorization Error (Error 400: invalid_request)* with t
 
 Hint: run ``az account subscription list`` to get a list of subscription IDs under your account.
 
+**Lambda**
+
+Create the file :code:`~/.lambda_cloud/lambda_keys`, go to `API Keys <https://cloud.lambdalabs.com/api-keys>`_ to generate a key, and then add the line
+
+.. code-block:: text
+
+  api_key = your_api_key_here
+
+to :code:`~/.lambda_cloud/lambda_keys`.
```

Comment:
How about this? Good to give direct instructions to users. (Please test if this actually works first :) )
 
```suggestion
Go to `API Keys <https://cloud.lambdalabs.com/api-keys>`_ to generate a key, and then add it to `~/.lambda_cloud/lambda_keys` by running:

.. code-block:: console
  $ # Create directories if required
  $ mkdir -p ~/.lambda_cloud
  $ # Add api_key = <your_api_key_here> line to lambda_keys file
  $ echo "api_key = <your_api_key_here>" > ~/.lambda_cloud/lambda_keys
```

Reply:
Nice! And I will test.

Reply:
Updated (and tested!)